snapcraft/yaml.py

```diff
@@ -272,10 +272,16 @@ def build_env_for_part(self, part, root_part=True):
             env += self.build_env_for_part(dep_part, root_part=False)
 
         if root_part:
+            # this has to come before any {}/usr/bin
+            env += _create_pkg_config_override(
+                part.bindir, part.installdir, stagedir)
             env += part.env(part.installdir)
             env += _runtime_env(stagedir)
             env += _runtime_env(part.installdir)
+            env += _runtime_env(part.ubuntu_unpackdir)
```

Comment:
@kyrofa python plugins fail to work, I'm thinking it might be due to this, did it work for you?